sa/model.go

```diff
@@ -400,7 +400,7 @@ type orderModelv2 struct {
 	Error                  []byte
 	CertificateSerial      string
 	BeganProcessing        bool
-	CertificateProfileName string
+	CertificateProfileName *string
```

Comment:
There should be a struct comment to match similar wording on the `crlShardModel` struct.
```
// CertificateProfileName is a pointer because it is a NULL-able column.
```